### PR TITLE
Hotfix#21

### PR DESCRIPTION
**Pull request by gnos28**

### CODE REVIEW

**Review by ClemDSC — APPROVED** [review]
ok!

File: frontend/src/components/FunFact.jsx [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+      <img src={`src/assets/${icon}`} alt={icon} />
       <div>
         {textFront}
         <p>
-          {factData}
-          {textBack}
+          {factData !== null && displayNumber1}
+          {factData !== null && textBack}
+          {factData2 !== null && displayNumber2}
+          {textBack2 !== null && textBack2}
```
Comment (lchastanet) [inline frontend/src/components/FunFact.jsx:51]: First condition seems useless

File: frontend/src/styles/Funfactlist.scss [hunk 2]
```diff
@@ -6,16 +6,18 @@
   width: 60%;
   .funfact {
     height: 5rem;
-    background-color: rgba(254, 254, 255, 0.6);
+    background-color: rgb(254, 254, 255);
+    filter: opacity(0.7);
+    transition: filter 0.4s ease-in-out;
     border-radius: 7px;
-    text-align: center;
+    // text-align: center;
```
Comment (lchastanet) [inline frontend/src/styles/Funfactlist.scss:13]: Commented code here

**Review by lchastanet — CHANGES_REQUESTED** [review]

File: frontend/src/components/FunFact.jsx (hunk 1 above)
Comment (gnos28) [reply frontend/src/components/FunFact.jsx]: corrected in last commit

File: frontend/src/styles/Funfactlist.scss (hunk 2 above)
Comment (gnos28) [reply frontend/src/styles/Funfactlist.scss]: corrected in last commit

**Comment by gnos28** [comment]
change request done in commit [7561bc2](https://github.com/WildCodeSchool/RMT-2203-TeamKarma-lions/pull/53/commits/7561bc28dd6df82f6407710330eb2305dc564abb)

**Review by lchastanet — APPROVED** [review]
Top !

**Pull request merged**